Dhrutham-UI-Documentation/CSS/dhrutham.css

```diff
@@ -0,0 +1,725 @@
+
+.card-vertical{
+    display:flex; 
+    flex-direction:column;
+    justify-content: space-between;
+}
+
+.card-shadow{
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+}
+
+.card-content-padding{
+    padding:var(--padding-1);
+}
+
+.card-header{
+    display: flex;
+}
+
+.card-text{
+    font-size:var(--font-size-text);
+    margin-bottom:0.3rem;
+}
+
+.card-title{
+    font-weight: 550;
+    font-size:var(--font-size-title);
+    margin-bottom:0.3rem;
+}
+
+.card-icon-topRight{
+    position: absolute;
+    top:0;
+    right:0;
+}
+
+.card-img{
+    width:100%;
+}
+
+.card-horizontal{
+    display:flex; 
+    justify-content: space-between;
+}
+
+.text-overlay{
+    position: absolute;
+    bottom: 0;
+    background-color:var(--white-color);
+    opacity: 0.7;
+    padding: 1rem;
+    width: 90%;
+    height: 1rem;
+    text-align: center;
+}
+
+.qty-div{
+    display: inline-block;
+    border:1px solid gray;
+    width:2rem;
+    padding:0.5px 0;
+    text-align: center;
+    margin:0 0.5rem;
+}
+/** Responsive-images **/
+
+.responsive-img{
+    width:100%;
+    height:auto;
+}
+
+.round-img{
+    border-radius: 50%;  
+}
+
+.img-margin{
+    margin-right:var(--padding-1);
+}
+
+/** Text Input **/
+
+.text-input{
+    height: 25px;
+    width:90%;
+    padding:2px 5px;
+    border:1px solid var(--gray-color);
+    border-radius:3px;
+    font-size:14px;
+    margin-top:3px;
+    margin-bottom:1rem;
+}
+
+.text-input:focus{
+    outline-width:thin;
+    outline-color:#60A5FA;
+}
+
+.flex-column{
+    display: flex;
+    flex-direction: column;
+}
+
+.border-bottom{
+    border:0;
+    border-bottom:1px solid;
+  }
+  
+.black-border{
+    border-color:black;
+  }
+  
+.gray-border{
+    border-color:var(--gray-color);
+  }
+  
+
+/** List **/
+.spaced-list-item-group{
+    list-style-type:none;
+}
+
+.spaced-list-item{
+    padding:var(--padding-1);
+    font-size: var(--font-size-text);
+}
+
+.spaced-list-item:hover{
+    background-color:#E5E7EB;
+    cursor: pointer;
+}
+
+/**Stacked List -notification**/
+
+.stacked-list-group{
+    border:1px solid var(--gray-color);
+}
+
+.stacked-item{
+    display: flex;
+    border-bottom:1px solid var(--gray-color);
+    padding:var(--padding-1); 
+    align-items: center;
+}
+
+/** Grid **/
+
+.grid-col-2 {
+    display: grid;
+    grid-template-columns: auto auto;
+    grid-column-gap: 5rem;
+    grid-row-gap: 2rem;
+  }
+  .grid-col-3 {
+    display: grid;
+    grid-template-columns: 22% 22% 22% 22%;
+    grid-column-gap: 1rem;
+    grid-row-gap: 2rem;
+  }
+  
+  @media (min-width: 0) {
+    .grid-col-2 {
+      display: grid;
+      grid-template-columns: 90%;
+    }
+    .grid-col-3 {
+      display: grid;
+      grid-template-columns: 50% 50%;
+    }
+  }
+  @media (min-width: 576px) {
+    .grid-col-2 {
+      display: grid;
+      grid-template-columns: auto;
+    }
+    .grid-col-3 {
+      display: grid;
+      grid-template-columns: 50% 50%;
+    }
+  }
+  @media (min-width: 840px) {
```

Comment:
Your media queries are overriding each other.
Use `max-width` to set rules for width less than or equal to the specified width.
Use `min-width` to set rules for width greater than or equal to the specified width.